### PR TITLE
Django Tutorial Part 6: Generic list and detail views

### PR DESCRIPTION
**Pull request by rzngnam1402**
ready

### CODE REVIEW

**Comment by rzngnam1402** [comment]
ready

File: locallibrary/catalog/templates/base_generic.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <html lang="en">
   <head>
     {% block title %}
-    <title>Local Library</title>
+    <title style="margin-top:20px">Local Library</title>
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/base_generic.html]: move style vào file css nhé

File: locallibrary/catalog/templates/catalog/author_detail.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% extends "base_generic.html" %}
+{% load i18n %}
+
+{% block content %}
+    <h1 style="margin-top:20px">{% trans "Author" %}: {{ author.first_name }} {{ author.last_name }}</h1>
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/catalog/author_detail.html]: tương tự, define style tag trong file css

**Comment by rzngnam1402** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
mợt pull e nhé

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**